common/include/pcl/common/gaussian.h

```diff
@@ -39,10 +39,9 @@
 
 #pragma once
 
-#include <sstream>
 #include <pcl/common/eigen.h>
 #include <pcl/point_cloud.h>
-#include <boost/function.hpp>
+#include <sstream>
```

Comment:
While rearranging includes, please separate them into blocks with empty lines. Otherwise clang-format will mix them (when we enable this in future).